tmk_core/common/action_util.c

```diff
@@ -290,6 +290,32 @@ void set_macro_mods(uint8_t mods) { macro_mods = mods; }
 void clear_macro_mods(void) { macro_mods = 0; }
 
 #ifndef NO_ACTION_ONESHOT
+/** \brief get oneshot mods
+ *
+ * FIXME: needs doc
+ */
+uint8_t get_oneshot_mods(void) { return oneshot_mods; }
+
+void add_oneshot_mods(uint8_t mods) {
+    if (oneshot_mods != mods) {
```

Comment:
```suggestion
    if ((oneshot_mods & mods) != mods) {
```
This condition should be changed otherwise `oneshot_time` will be updated and `oneshot_mods_changed_kb` will be called even if none mod is actually set.

Reply:
Good catch!